code/modules/power/apc.dm

```diff
@@ -58,7 +58,7 @@
 	var/damage_deflection = 10
 	resistance_flags = FIRE_PROOF
 	armor = ARMOR_VALUE_MEDIUM
-	req_access = list(ACCESS_ENGINE_EQUIP, ACCESS_BOS)
+	req_access = list(ACCESS_ENGINE_EQUIP, , ACCESS_TOWN_SCIENCE)
```

Comment:
The bos has apcs in there base, unless they are specific ones.

Reply:
> The bos has apcs in there base, unless they are specific ones.

The basement clinic has APCs too